### PR TITLE
Add Initial Support for VMware SD-WAN / Velocloud

### PR DESCRIPTION
**Pull request by Nocturr**
Adding Initial Support for VMware SD-WAN / Velocloud gateways

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](https://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.
- [x] If my Pull Request makes discovery/polling/yaml changes, I have added/updated [test data](https://docs.librenms.org/Developing/os/Test-Units/).

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by Jellyfrog** [comment]
I wonder what the OS name should be.
>VMware SASE™ (formerly known as VeloCloud) 

sdwan, sase, velocloud?

**Comment by Nocturr** [comment]
> I wonder what the OS name should be.
> 
> > VMware SASE™ (formerly known as VeloCloud)
> 
> sdwan, sase, velocloud?

AS far as I know, SASE is the cloud-hosted part of the product.
The appliances and vm's you place on prem are called 'VeloCloud Edge' still, but VeloCloud is the product they bought what will slowly traverse to 'VMware SD-WAN'.
The appliances run a custom Linux operating system, and a name for it is not known to me and my VMware partner. That's why I called it VMware SD-WAN

File: tests/data/vmware-sdwan.json [hunk 1]
```diff
@@ -0,0 +1,2571 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.8072.3.2.10",
+                    "sysDescr": "VeloCloud EDGE610",
+                    "sysContact": "<private>",
+                    "version": null,
```
Comment (Jellyfrog) [inline tests/data/vmware-sdwan.json:10]: No version number or build number or sth to populate this with?

File: tests/data/vmware-sdwan.json [hunk 2]
```diff
@@ -0,0 +1,2571 @@
...
+                    "hardware": "EDGE610",
+                    "features": null,
+                    "os": "vmware-sdwan",
+                    "type": "appliance",
+                    "serial": null,
```
Comment (Jellyfrog) [inline tests/data/vmware-sdwan.json:15]: Same as above, but doubt it?

File: tests/data/vmware-sdwan.json (hunk 1 above)
Comment (Nocturr) [reply tests/data/vmware-sdwan.json:10]: No, they don't expose the version of the software via snmp..

File: tests/data/vmware-sdwan.json (hunk 2 above)
Comment (Nocturr) [reply tests/data/vmware-sdwan.json:15]: Same here, no serial via snmp.. can provide the raw snmpwalk if needed, but sadly they don't provide that data.

File: tests/data/vmware-sdwan.json (hunk 1 above)
Comment (Jellyfrog) [reply tests/data/vmware-sdwan.json:10]: No worries

**Comment by Jellyfrog** [comment]
@librenms/core thoughts on OS name?

**Comment by murrant** [comment]
My vote is os name: velocloud
os text: VMware SD-WAN

File: mibs/vmware-sdwan/VELOCLOUD-MIB [hunk 3]
```diff
@@ -0,0 +1,42 @@
+-------------------------------
```
Comment (murrant) [inline mibs/vmware-sdwan/VELOCLOUD-MIB:1]: Please put this MIB in the vmware directory and add `mib_dir: vmware` on the OS.

Comment (murrant) [reply mibs/vmware-sdwan/VELOCLOUD-MIB:1]: MIB directories are per-vendor, not per-os.

**Review by murrant — APPROVED** [review]
Looks good, nice work.

**Pull request merged**